app/screens/conversion-flow/use-convert-money-details.ts

```diff
@@ -0,0 +1,95 @@
+import { DisplayCurrency, MoneyAmount, WalletOrDisplayCurrency } from "@app/types/amounts"
+import React from "react"
+import { usePriceConversion } from "../../hooks/use-price-conversion"
+import { WalletCurrency } from "@app/graphql/generated"
+
+type Wallet = {
```

Comment:
shouldn't this be from generated.ts?

you can trim down the type as needed, something like `Pick<Wallet, ["id", "balance", "walletCurrency"]>` 

not sure about the syntax but you'll get the idea: referencing the typescript help understand that those contexts are tied together

Reply:
Didn't realize you could do the pick. I will make that change.